hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -111,7 +116,7 @@ protected void doRefresh() {
     String metadataLocation = null;
     try {
       Table table = metaClients.run(client -> client.getTable(database, tableName));
-      validateTableIsIceberg(table, fullName);
+      validateTableIsIceberg(table, fullName());
```

Comment:
Is this change needed? This is an instance variable so there should be no need to call the getter.

Reply:
Reverted